ci/install-upstream-wheels.sh

```diff
@@ -58,6 +58,8 @@ python -m pip install \
     --upgrade \
     --no-build-isolation \
     git+https://github.com/pydata/bottleneck
+# manually install pint deps. These will be pulled in by the next released version, but for now we have to install them (since we use `--no-deps` pip will not pull them in the install).
+python -m pip install --upgrade 'flexcache>=0.3' 'flexparser>=0.3' 'appdirs>=1.4.4' typing_extensions
```

Comment:
No big change but possibly this could just be `python -m pip install --upgrade pint`, and that will do deps?

Reply:
that would work if the released version had the same dependencies as the upstream-dev version. Unfortunately, all of these (except `typing_extensions`) have been added since the last release.

Another option would be to add `python -m pip install --upgrade git+https://github.com/hgrecco/pint` (i.e. no `--no-deps` for `pint`)

Reply:
now that `pint=0.24` has been released, we can actually use `pip` to figure out the dependencies.